src/features/hotspots/setup/HotspotSetupLocationInfoScreen.tsx

```diff
@@ -22,6 +23,10 @@ const HotspotSetupLocationInfoScreen = () => {
     }
   }
 
+  const skipLocationAssert = async () => {
```

Comment:
probably doesn't matter, but could wrap this in a `useCallback`

Reply:
Do we need it for things that dont use any state or anything? Would rather keep it simple if it doesn't really affect anything